packages/manager/cypress/support/plugins/split-run.ts

```diff
@@ -27,11 +30,111 @@ export const splitCypressRun: CypressPlugin = (_on, config) => {
     const totalRunners = parseInt(splitRunTotalRunners, 10);
     const runner = parseInt(splitRunRunnerIndex, 10);
 
-    // Override configuration spec pattern to reflect test subset for this runner.
+    // Override configuration spec pattern to reflect test subset for this runner...
     const specs = globSync(config.specPattern);
-    // Sort spec filenames deterministically.
-    // Or at least as deterministically as we can in a pinch...
-    specs.sort((a: string, b: string): number => {
+
+    // Attempt to locate a weights file for the test suite being run.
+    // const weightsPath = `cypress/weights/${suiteName}-weights.json`;
+
+    // Information about test weights to be included in the info table, if applicable.
+    // TODO type.
+    let weightsInfo: Record<string, any> = {
```

Comment:
The type here isn't super critical since this is just for informative purposes
```suggestion
    let weightsInfo: Record<string, string | number> = {
```